plugins/modules/ec2_vpc_route_table.py

```diff
@@ -256,6 +256,11 @@
           returned: when the route is via a NAT gateway
           type: str
           sample: local
+        carrier_gateway_id:
+          description: ID of the Carrier gateway.
+          returned: when the route is via a Carrier gateway
+          type: str
+          sample: local
```

Comment:
```suggestion
          sample: local
          version_added: 6..0.0
```

Reply:
@tremble is that syntax correct for `version_added`? or should I set to `6.0.0`?

Reply:
6.0.0, typo on my side :)